extensions/amp-story/1.0/amp-story.js

```diff
@@ -654,6 +669,21 @@ export class AmpStory extends AMP.BaseElement {
       }
     });
 
+    if (isExperimentOn(this.win, 'amp-story-branching')) {
+      this.win.addEventListener('hashchange', () => {
+        const maybePageId = parseQueryString(this.win.location.hash)['page'];
+        if (this.isActualPage_(maybePageId)) {
+          this.switchTo_(maybePageId, NavigationDirection.NEXT).then(() => {
+            this.closeOpacityMask_();
+          });
+          // Remove the fragment parameter from the URL
+          const {history} = this.win;
+          history.pushState(
+              '', this.win.document.title, this.win.location.pathname);
```

Comment:
Can you add a comment to document the first parameter?

Reply:
Done, and for the other args as well.